torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -702,6 +702,17 @@ def set_state_dict_type(
 
     @staticmethod
     def get_state_dict_type(module: nn.Module) -> StateDictSettings:
+        """
+        Get the state_dict_type and the corresponding configurations
+        for the FSDP modules rooted at ``module``. The target module
+        does not have to be an FSDP module. Note that this method will
```

Comment:
nit: Should we just document via `Raises:` in the docstring the condition for raising an error?